src/actions/vstudio/_vstudio.lua

```diff
@@ -340,7 +340,13 @@
 		-- Then the system libraries, which come undecorated
 		local system = config.getlinks(cfg, "system", "fullpath")
 		for i = 1, #system do
-			table.insert(links, path.appendextension(system[i], ".lib"))
+			-- Add extension if required
+			local link = system[i]
+			if link:match("[^.]+$") == link then
```

Comment:
Yup, and that's probably the reason a fix hasn't been submit before ;)
This is just for windows, so I think it's safe to assume `.lib` and `.obj` for now.